src/Avalonia.Controls.ColorPicker/ColorView/AlphaComponentPosition.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Avalonia.Controls
+{
+    /// <summary>
+    /// Defines the position of a color's alpha component relative to all other components.
+    /// </summary>
+    public enum AlphaComponentPosition
```

Comment:
Naming here was debated:

 * Orignially `AlphaPosition` but that is considered a bit too generically worded
 * An alternative `ColorAlphaPosition` is possible but perhaps "component" is better for what it does
 * Using `AlphaComponentPosition` matches with the `RgbComponent` and `HsvComponent` terminology